Sources/OpenCombine/Publishers/Publishers.Filter.swift

```diff
@@ -0,0 +1,213 @@
+//
+//  Publishers.Filter.swift
+//  
+//
+//  Created by Joseph Spadafora on 7/3/19.
+//
+
+import Foundation
```

Comment:
We don’t want to depend on Foundation, let’s remove this import

Reply:
Ooops, I think that came with the boilerplate.  Good catch